terraform/s3.tf

```diff
@@ -0,0 +1,13 @@
+provider "aws" {
+  region = "eu-west-2"
+}
+
+resource "aws_s3_bucket" "c14-bandcamp-reports" {
+  bucket = "c14-bandcamp-reports"
+  acl    = "private"
```

Comment:
Does this mean the S3 is private? Will that work with everything else being public?

Reply:
yes I tested with boto3 and was able to download a file from the s3 bucket.

Reply:
just tested with uploading a file and that worked too